src/components/MultiChoiceResult.vue

```diff
@@ -4,9 +4,13 @@
       <strong v-if="locale == undefined">{{ data.title }}</strong>
       <strong v-if="locale !== undefined">{{ data.titleData[locale] }}</strong>
       <br />
-      <ul v-for="(str, index) in data.value" :key="index">
-        <li>{{ getItemLabel(str, index) }}</li>
-      </ul>
+      <div v-for="(str, index) in data.value" :key="index">
```

Comment:
Aligns this component with MultiChoiceValueResult. Fixes the bug, and gives a consistent look and code.